MacOSX/opensc-uninstall

```diff
@@ -32,10 +32,15 @@ rm -f  /Library/LaunchAgents/pkcs11-register.plist
 rm -f  /Library/LaunchAgents/opensc-notify.plist
 rm -rf /System/Library/Security/tokend/OpenSC.tokend
 
+# Unload launchagents
+launchctl remove pkcs11-register
+launchctl remove opensc-notify
```

Comment:
I think this needs to be run as the logged in user instead of `root`, see [the install script](https://github.com/OpenSC/OpenSC/blob/master/MacOSX/scripts/postinstall#L39)

Reply:
I just realized, that we've installed the services in a single place (`/Library/LaunchAgents`), so removing this once is probably enough. We'll see if we are getting an issue with this.